stepmania.nsi

```diff
@@ -410,6 +410,7 @@ Section "Main Section" SecMain
 
 	; remove and install themes
 	RMDir /r "$INSTDIR\Themes\_fallback"
+	RMDir /r "$INSTDIR\Themes\Til Death"

```

Comment:
Can you make a separate commit for this? I don't want to do this here because it's something that needs to be discussed further.